website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx

```diff
@@ -0,0 +1,37 @@
+import { ChangeEvent, useState } from 'react';
+import { useField } from 'formik';
+import { Input, Checkbox } from '@gear-js/ui';
+
+import styles from './SwitchableField.module.scss';
+
+type Props = {
+  name: string;
+  label: string;
+  defaultValue: string;
+};
+
+const SwitchableField = (props: Props) => {
```

Comment:
```suggestion
const PayloadType = (props: Props) => {
```
I don't think that it's reusable component, so specific name is fine

Reply:
It won't make it any worse.